samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -157,13 +158,41 @@ static void publish_telemetry() {
 }
 
 /* Include the sample.  */
-bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
+static bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
     printf("Starting App Version %s\r\n", APP_VERSION);
     IotConnectClientConfig *config = iotconnect_sdk_init_and_get_config();
     azrtos_config.ip_ptr = ip_ptr;
 	azrtos_config.pool_ptr = pool_ptr;
 	azrtos_config.dns_ptr = dns_ptr;
 
+#ifdef IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+    static char iotconnect_cpid[64] = {0};
+    static char iotconnect_env[64] = {0};
```

Comment:
I would make those variables statics in the file scope at the top. That's where we are keeping all of the rest of "permenent" variables. Also I am not 100% sure if static variables in function scope would accept the {0} initializer. I think I ran into some issue with one of the other boards compilers so I had to memset the values manually. You really don't even need to memset them. Just set iotconnect_duid[0] = 0 and it will be a blank string. The rest of the array doesn't matter.

Reply:
Done.
The "= {0};" was due to seeing some input strings not NULL terminating correctly.